source/android/Renaissance/lib/http_services.dart

```diff
@@ -255,6 +255,58 @@ class HttpService {
     }
   }
 
+  Future<Tag> getTopic(String topic_id) async {
+    String url = baseUrl + '/topic/get-topic/$topic_id/';
+    Response res = await get(Uri.parse(url), headers: headers);
+
+    if (res.statusCode == 200) {
+      var temp = jsonDecode(res.body);
+      /*List<Milestonee> milestoness = [];
+      List<Topic> topicss = [];
+
+      (temp["milestones"]).map((tag) {
+        milestoness.add(Milestonee.fromJson(tag));
+      }).toList();
+
+      (temp["topics"]).map((tag) {
+        topicss.add(Topic.fromJson(tag));
+      }).toList();*/
```

Comment:
You can get rid od commented code blocks here.

Reply:
of*

Reply:
Done